issue_tracker/models.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib.auth import get_user_model
+from django.db import models
+
+User = get_user_model()
+
+
+class Issue(models.Model):
+    title = models.CharField(max_length=50, help_text="Type of exception")
+    url = models.CharField(max_length=50, help_text="Url endpoint")
+    headers = models.TextField()
+    body = models.TextField()
+    method = models.CharField(max_length=10, help_text="Type of http method")
+    ip_address = models.GenericIPAddressField(blank=True, null=True)
+    response = models.TextField(help_text="Error log")
+    description = models.TextField(help_text="Reason of error")
+    status_code = models.CharField(max_length=10, blank=True, null=True)
+    user = models.ForeignKey(User, null=True, on_delete=models.DO_NOTHING)
+    content_type = models.CharField(max_length=50)
```

Comment:
Content type should be not GenericField ?

Reply:
Yes, I'm removing this from models as we already store the content type inside the header payload.